.github/workflows/turing.yaml

```diff
@@ -543,6 +589,12 @@ jobs:
           KUBECONFIG_USE_LOCAL: true
         run: go test -v -parallel=2 ./e2e/... -tags=e2e -run TestEndToEnd
 
+      # Invoke helm hooks on delete
+      - name: Tear down infrastructure job
+        run: |
+          helm delete -n infrastructure turing-init --timeout 15m
+          kubectl get po -A
```

Comment:
If it's a code, that's expected to be read and maintained by multiple people, it's advised to use the long form of flags for better readability, i.e 
```
kubectl get pods --all-namespaces
```

Reply:
ok, done